src/lib/plugins/placeBlock.js

```diff
@@ -16,7 +16,7 @@ module.exports.server = (serv, { version }) => {
   const itemPlaceHandlers = new Map()
   serv.placeItem = (data) => {
     const handler = itemPlaceHandlers.get(data.item.type)
-    return handler ? handler(data) : { id: data.item.type, data: data.item.metadata }
+    return handler ? handler(data) : (serv.supportFeature('theFlattening') ? {} : { id: data.item.type, data: data.item.metadata })
```

Comment:
This is wrong

Reply:
I don't think I understand what you mean.

Reply:
why would we return nothing after the flattening ?

Reply:
placeItem is currently only used in the same file, handling the block_place packet from clients. It's destructured into id and data after being called, then the id is used to find the corresponding blocks. Blocks with the same ID as an item is not the corresponding block for the item when the item does not have a corresponding block, so I return an empty object with ID undefined so the wrong block won't be placed.

Reply:
In pre-flattening times this can be used as a fallback for items that somehow don't have an item place handler as the old assumption is indeed true.